src/Campaigns/resources/admin/components/CampaignDetailsPage/CampaignDetailsPage.module.scss

```diff
@@ -118,6 +119,34 @@
     font-weight: 600;
 }
 
+.status {
```

Comment:
Now that we are using the <StatusBadge /> component again, we can remove this CSS class.

Reply:
Resolved: https://github.com/impress-org/givewp/pull/7565/commits/6ef588dac2c50f5fcf2c5663ecbc8b3cb9979326